variables.tf

```diff
@@ -98,16 +98,16 @@ variable "tables" {
   description = "A list of objects which include table_id, table_name, schema, clustering, time_partitioning, range_partitioning, expiration_time and labels."
   default     = []
   type = list(object({
-    table_id    = string,
-    description = optional(string),
-    table_name  = optional(string),
-    schema      = string,
-    clustering  = list(string),
+    table_id                 = string,
+    description              = optional(string),
+    table_name               = optional(string),
+    schema                   = string,
+    clustering               = list(string),
+    require_partition_filter = optional(bool),
```

Comment:
It would be good to add an upgrade guide noting this change

https://github.com/terraform-google-modules/terraform-google-bigquery/tree/master/docs

Reply:
Ack